internal/pkg/providers/aws/securityGroupFetcher.go

```diff
@@ -0,0 +1,95 @@
+package aws
```

Comment:
Perhaps rename class to `securityGroup.go`?

Reply:
It can be securitygroup_fetcher.go and securitygroup_updater.go

Reply:
@waseem-h will set it to security_group_fetcher.go since security group is not one word

Reply:
https://github.com/golang/go/issues/36060
@waseem-h why use "_" I see that "xxx_test" makes sense but why camel is not right?

Reply:
> Go source file names should be lowercase_with_underscores.go and not camelCase.go or TitleCase.go. This makes them consistent with other language conventions, like _test.go for test files, or _windows.go for build constraints. It also prevents ambiguities on case-insensitive file systems.

https://github.com/golang/go/issues/36060

will change other file names from camel case to this "_" notation